help.go

```diff
@@ -77,10 +78,25 @@ var helpCommand = &Command{
 			HelpPrinter(cCtx.App.Writer, templ, cCtx.Command)
 			return nil
 		}
+
+		// Case 6, handling incorporated in the callee itself
 		return ShowSubcommandHelp(cCtx)
 	},
 }
 
+func anyFlag(candidates []string, flags []Flag) bool {
+	for _, flag := range flags {
+		for _, alias := range flag.Names() {
+			for _, c := range candidates {
+				if c == alias {
+					return true
+				}
+			}
+		}
+	}
+	return false
+}
+
```

Comment:
I can't find where this function is used 🤔

Reply:
Oh, that should never have made it into the PR.
It was a different approach to solving the problem. Thanks for spotting this Dan!

Pushed a corrective commit that removes this function, and also the redundant test which DC pointed out.
Once you're happy, I'll squash the PR.